src/MagicWormhole/Internal/ClientProtocol.hs

```diff
@@ -65,37 +66,36 @@ receiveEncrypted conn key = do
 
 -- | Encrypt a mailbox message, deriving the key from the phase.
 encryptMessage :: Connection -> SessionKey -> Messages.Phase -> PlainText -> IO Messages.Body
-encryptMessage conn key phase plaintext = Messages.Body <$> encrypt derivedKey plaintext
+encryptMessage conn key phase plaintext = Messages.Body . unCipherText <$> encrypt derivedKey plaintext
```

Comment:
`unCipherText` is a function that takes a `CipherText` and returns the `unCipherText` field, right?  This helps with my interpretation of what the `unCipherText` field of `CipherText` is - the byte sequence holding the cipher text.  The `un...`-prefix naming scheme wasn't immediately obvious to me.  `unCipherText` to me suggests plaintext but for some subjective reason `unPlainText` does not suggest ciphertext.

Not sure this is an actionable comment.  Unless you can think of more evocative field names.  Or maybe just some comments.

Reply:
Yes, `unCipherText` is as you describe. Its type signature in full is

```haskell
unCipherText :: CipherText -> Text
```

I can see why the name could imply that it somehow gets you the plain text.

```haskell
newtype Foo a = Foo { unFoo :: a }
```

Is kind of a Haskell idiom. Probably borne out of a lack of namespacing for fields (we can't just call them all `original` or `underlying`). 

I'll add some comments.

Reply:
Actually, `getCipherTextBytes` or `cipherTextToByteString` would both be clearer names. I'll rename when I'm at an appropriate computer.